core/test/financial-templates/Liquidatable.js

```diff
@@ -91,6 +90,10 @@ contract("Liquidatable", function(accounts) {
   };
 
   beforeEach(async () => {
+    // Force each test to start with a simulated time that's synced to the startTimestamp.
+    const timer = await Timer.deployed();
+    await timer.setCurrentTime(startTime);
```

Comment:
This is a better solution to creating a new Timer, to instead just set the time at the beginning of each test.